src/index.jsx

```diff
@@ -21,7 +22,18 @@ import { OrdersAndSubscriptionsPage } from './orders-and-subscriptions';
 
 import './index.scss';
 
+mergeConfig({
```

Comment:
Can you keep the temp comment removed from `service.js`: `"Temporary: until we add COMMERCE_COORDINATOR_BASE_URL in frontend-platform, use mergeConfig to join it..."`? This is work done for Theseus, which could also apply to Ponderosa if we want to be able to get these URLs from `getConfig()`, post MVP.

Reply:
I'm assuming `COMMERCE_COORDINATOR_BASE_URL` is still accessible from the `service.js` file you moved it from (we're not breaking that logic)?

Reply:
> Can you keep the temp comment

Added with a modification to the comment,

> (we're not breaking that logic)?

Yes we aren't breaking any logic